apps/e2e/cypress/e2e/FAPs.cy.ts

```diff
@@ -2233,7 +2236,45 @@ context('Fap meeting components tests', () => {
 
       cy.finishedLoading();
 
-      cy.get('[aria-label="Submit instrument"] button').should('be.disabled');
+      cy.get('[aria-label="Detail panel visibility toggle"]').should('exist');
+
+      cy.get('button[aria-label="Submit instrument"]').should('not.exist');
+      cy.get('button[aria-label="Unsubmit instrument"]')
+        .should('exist')
+        .and('not.be.disabled');
+
+      cy.get('[aria-label="Detail panel visibility toggle"]').first().click();
+
+      cy.get('[aria-label="Drag proposals to reorder"]').should('not.exist');
+
+      // NOTE: Trying to catch the failure of cy.reorderFapMeetingDecisionProposals because instrument is submitted
+      cy.on('fail', (err) => {
```

Comment:
I think I have tested this one with and without prevention of reordering. I added it just because of that, to catch this if something changes in the backend code. But thanks for mentioning it and will test it once more to verify your point.